config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java

```diff
@@ -208,7 +184,7 @@ public int getDispatchPort() {
     }
 
     public int getHttpPort() {
-        return getRelativePort(4);
+        return getRelativePort(2);
```

Comment:
Port number changes might cause issues during version upgrade, where some nodes run with old config and some nodes run with new config.  e.g. health check trying to use wrong port will see node as down.